app/main.py

```diff
@@ -2,0 +1,57 @@
+class SoftwareEngineer:
+    def __init__(self, name: str) -> None:
+        self.name = name
+        self.skills = []
+
+    def learn_skill(self, skill: str) -> None:
+        self.skills.append(skill)
```

Comment:
Consider using `extend()` or `+=` instead of `append()` in the `learn_skill` method if you plan to add multiple skills at once. This was a suggestion from the previous review.